src/findings.ts

```diff
@@ -0,0 +1,55 @@
+import { Finding, TransactionEvent, FindingSeverity, FindingType } from "forta-agent";
+import { ethers } from "ethers";
+
+export const createFinding = (
+  callSignature: string,
+  txEvent: TransactionEvent,
+  args: ethers.utils.Result
+): Finding | null => {
+  let name, description, alertId, metadata;
+
+  if (callSignature.includes("create")) {
+    name = "CREATE Agent Detected";
+    description = "CREATE function called by Nethermind's deployer address";
+    alertId = "NETHERMIND-1";
+    metadata = {
+      agentId: args[0].toString(),
+      owner: args[1],
+      metadata: args[2],
+      chainIds: args[3].join(", "),
+    };
+  } else if (callSignature.includes("update")) {
+    name = "UPDATE Agent Detected";
+    description = "UPDATE function called by Nethermind's deployer address";
+    alertId = "NETHERMIND-2";
+    metadata = {
+      agentId: args[0].toString(),
+      metadata: args[1],
+      chainIds: args[2].join(", "),
+    };
+  } else if (callSignature.includes("disable")) {
+    name = "DISABLE Agent Detected";
+    description = "DISABLE function called by Nethermind's deployer address";
+    alertId = "NETHERMIND-3";
+    metadata = {
+      agentId: args[0].toString(),
+      reason: args[1].toString(),
+    };
+  } else {
+    return null;  // Ignorieren Sie unbekannte Events
+  }
```

Comment:
```suggestion
  } else {
    name = "DISABLE Agent Detected";
    description = "DISABLE function called by Nethermind's deployer address";
    alertId = "NETHERMIND-3";
    metadata = {
      agentId: args[0].toString(),
      reason: args[1].toString(),
    };
  }
```
Could replace one of the `if` checks and have its logic in the `else` branch. Since `filterFunction` would only return instances of the specific function calls it is filtering for, you could safely use `else` like that.

Also, you could use `call.name` instead of `call.signature`.

P.S. Who would've thought I'd be learning German doing PR reviews. 😝

Reply:
Seems like i forgot to remove that comment ;)

Sometimes i put german ones during development but try to remove/translate before committing haha xD